models/repo/repo.go

```diff
@@ -277,9 +277,9 @@ func (repo *Repository) AfterLoad() {
 	repo.NumOpenProjects = repo.NumProjects - repo.NumClosedProjects
 	repo.NumOpenActionRuns = repo.NumActionRuns - repo.NumClosedActionRuns
 
-	// this is a temporary behaviour to support old repos, next step is to store the object format in the database
-	// and read from database so this line could be removed. To not depend on git module, we use a constant variable here
-	repo.ObjectFormatName = "sha1"
+	if repo.ObjectFormatName == "" {
+		repo.ObjectFormatName = "sha1"
+	}
```

Comment:
And this lines should be removed after the migrated added.

Reply:
That's a good point. Did you have any success in migrating the migration to use xorm functionality like `xorm.Sync()` or are we sticking with the manual method?

Reply:
130e8bf935d44364cec62de8b071cefc85d8fa8b